src/module_1/module_1_meteo_api.py

```diff
@@ -137,7 +132,7 @@ def plotting(list):
     fig.tight_layout()
     # plt.show()
     ax1.legend(handles=[p1, p2, p3])
-    plt.savefig("mygraph.png")
+    plt.savefig(city + ".png")
 
 
 def create_labels(text, list):
```

Comment:
- Consider the use of "f-strings"
- Splitting the last 4 characters may be dangerous for the futue. You could have use `split` or even better, a regex